index.js

```diff
@@ -589,4 +589,18 @@ export const mentorshipTeam = [
 			},
 		],
 	},
+	{
+		name: 'Hai Anh',
+		title: 'Student @ Fordham University',
+		hobbies: 'walking, bike riding, road trip',
+		avatar:
+			'https://www.linkedin.com/in/hennessy-nguyen-18a8a8163/',
```

Comment:
let's update to `https://res.cloudinary.com/khoa165/image/upload/v1711768766/viettech/haianh.jpg`